app.js

```diff
@@ -19,9 +18,20 @@ var app = express();
 app.set('views', path.join(__dirname, 'views'));
 app.set('view engine', 'ejs');
 
+// Authentication 
+app.use(session({
+  secret: 'secret',
```

Comment:
plaintext AND openSource secret.
Security threat. Will address in #15

Reply:
I was intending to put this in environment variables